ios/RNCWKWebView.m

```diff
@@ -91,6 +71,11 @@ - (instancetype)initWithFrame:(CGRect)frame
   return self;
 }
 
+- (void)dealloc
+{
+  [[NSNotificationCenter defaultCenter] removeObserver:self];
```

Comment:
What does this change do?

Reply:
Considering this: https://useyourloaf.com/blog/unregistering-nsnotificationcenter-observers-in-ios-9/

Reply:
Oh maybe this isn't required, but the init method ~10 lines above registers the RNCWKWebView class as observer. So I think its "good style" remove the observer (or any other listener, subscriptions) when the class was deallocated. I could remove the dealloc method if you think this could confuse someone. I would prefer to keep it.